roles/forklift_versions/molecule/default/verify.yml

```diff
@@ -7,7 +7,7 @@
   vars:
     forklift_versions_file: "{{ playbook_dir }}/fixtures/versions.yaml"
     pipeline_type: katello
-    pipeline_os: centos8-stream
+    pipeline_os: centos9-stream
```

Comment:
The tests below use Katello 4.8 (and 4.9, but I actually think that's pretty much duplicated by now and can be dropped) for testing whether "the right" versions are generated.
However, 4.8/4.9 is not available for EL9, so versions don't get set and the tests fail (rightfully so).
If you update the tests to use 4.12 (and drop the "4.8" one -- it was testing the "no standalone candlepin repo", which we don't have anymore), they should pass again (you gotta also update the assertions to foreman 3.10, candlepin 4.3, pulpcore 3.39)

Reply:
@evgeni, i can't understand what is wrong now, tests are still failing.

Reply:
Commented below where the actual failures are.

Reply:
@evgeni thanks, also it passed all the test now.